pkg/services/forwarder/ports.go

```diff
@@ -378,7 +379,10 @@ func tcpipAddress(nicID tcpip.NICID, remote string) (address tcpip.FullAddress,
 		return address, errors.New("invalid remote addr")
 	}
 
-	port, err := strconv.Atoi(split[1])
+	port, err := strconv.ParseUint(split[1], 10, 16)
```

Comment:
`ParseUint(…, 10, 16)` guarantees that `port` will fit in 16 bits, so the `port > math.MaxUint16` can be removed.